src/os/bluestore/BlueStore.cc

```diff
@@ -4155,8 +4155,19 @@ void BlueStore::MempoolThread::_resize_shards(bool interval_stats)
   for (auto i : store->onode_cache_shards) {
     i->set_max(max_shard_onodes);
   }
+  //sum is in data_used
+  uint64_t actual_trim = 0;
```

Comment:
unused

Reply:
and max_shard_buffer declaration above to be removed

Reply:
@ifed01 thanks!